docs/source/author/topics.rst

```diff
@@ -0,0 +1,551 @@
+        solution: binary-numbers/programming-exercises/exercise-1.1/scratch-solution.md
+      python:
+        hints: binary-numbers/programming-exercises/exercise-1.1/python-hints.md
+        solution: binary-numbers/programming-exercises/exercise-1.1/python-solution.md
+
+  count-to-a-million:
+    number: 2
+    difficulty-level: 3
+    md-file: binary-numbers/programming-exercises/exercise-2/index.md
+    learning-outcomes:
+      - programming-basic-logic
+    programming-languages:
+      scratch:
+        hints: binary-numbers/programming-exercises/exercise-2/scratch-hints.md
+        solution: binary-numbers/programming-exercises/exercise-2/scratch-solution.md
+      python:
+        hints: binary-numbers/programming-exercises/exercise-2/python-hints.md
+        solution: binary-numbers/programming-exercises/exercise-2/python-solution.md
+
+.. _learning-outcomes-file:
+
+Learning Outcomes File
+------------------------------------------------------------------------------
+
+This file defines the learning outcomes avilable for all topics.
+
+The location of this file is within the language folder.
+This file is listed in a :ref:`application-structure-file`.
+This file can contain as many learning outcomes as you like.
+
+The file should only contain pairs of outcome slug to outcome text pairs.
+For example:
+
+.. code-block:: yaml
+
+  binary-data-representation: Explain how a binary digit is represented using two contrasting values.
+  binary-count: Demonstrate how to represent any number between 0 and 31 using binary.
+  binary-convert-decimal: Perform a demonstration of how the binary number system works by converting any decimal number into a binary number.
+  binary-justify-representation: Argue that 0’s and 1’s are still a correct way to represent what is stored in the computer.
+
+.. note::
+
+  Lessons and programming exercises link to learning outcomes by listing
+  their slug.
+
+.. _curriculum-links-file:
+
+Curriculum Links File
+------------------------------------------------------------------------------
+
+This file defines the curriculum links avilable for all topics.
+
+The location of this file is within the language folder.
+This file is listed in a :ref:`application-structure-file`.
+This file can contain as many curriculum links as you like.
+
+The file should have the following key/value pairs:
+
+.. code-block:: yaml
+
+  # The URL slug for the curriculum link
+  maths:
+    # The display name for the curriculum
+    name: maths
+
+.. note::
+
+  Lessons and follow up activities link to curriculums by listing their
+  slug.
+
+An example curriculum links file with multiple curriculums may look like
+the following:
+
+.. code-block:: yaml
+
+  maths:
+    name: Maths
+
+  science:
+    name: Science
+
+  art:
+    name: Art
+
+.. _adding-topics-content:
+
+Adding topics content
+==============================================================================
+
```

Comment:
this section should be the first thing, not the last. If I am a returning author, I want to be able to quickly see the sequence of steps, not read all the docs all over again.